product-service/products.ts

```diff
@@ -0,0 +1,24 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import * as data from './productList.json';
+
+export const getProducts: APIGatewayProxyHandler = async (event) => {
+  console.log("getProducts", JSON.stringify(event.body));
+  const response = {
+    statusCode: 200,
+    headers: {
+      'Access-Control-Allow-Origin': '*',
+      'Access-Control-Allow-Credentials': true,
+    },
+    body: '',
+  };
+
+  try {
+    response.body = JSON.stringify(data, null, 2);
```

Comment:
Hi, i don't see where async/await is implemented. It's fine if it's not implemented, just want to be sure, since it's listed on your PR description.